.oxlintrc.json

```diff
@@ -1,5 +1,12 @@
 {
   "$schema": "./node_modules/oxlint/configuration_schema.json",
+  "jsPlugins": [
+    {
+      "name": "playwright",
+      "specifier": "eslint-plugin-playwright"
+    }
+    // eslint-plugin-react-hook-form won't work — all its rules use context.getScope()
```

Comment:
Is this still the case? I believe getScope() may be implemented on the most recent releases of oxlint? https://github.com/oxc-project/oxc/blob/e9d346f1f219025229013da47386d387823c220e/apps/oxlint/src-js/plugins/scope.ts#L359

Reply:
Thanks, putting it on my todo list.

Reply:
Thank you! Subscribed to that issue.